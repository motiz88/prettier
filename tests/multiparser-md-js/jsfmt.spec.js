run_spec(__dirname, { parser: "remark" });
