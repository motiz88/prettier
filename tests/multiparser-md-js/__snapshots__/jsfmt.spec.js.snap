// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`code-js.md 1`] = `
\`\`\`js
function
foo
() 
{

}
\`\`\`
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
\`\`\`js
function foo() {}
\`\`\`

`;

exports[`code-js-inner-fence.md 1`] = `
\`\`\`node
const a = "\\\`\\\`\\\`";
\`\`\`
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
\`\`\`node
const a = "\`\`\`";
\`\`\`

`;
